_posts/2019-06-21-formats.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: post
+title: OME's position regarding file formats
+categories: future-plans community file-formats
+---
+
+
+<!-- https://stackoverflow.com/questions/41481100/add-padding-or-margin-broke-float -->
+<!-- https://css-tricks.com/almanac/properties/q/quotes/ -->
+<style type="text/css">
+blockquote {
+  font-size: 160%;
+  margin-left: -3rem;
+  margin-right: -3rem;
+  padding-right: 1rem;
+  padding-left: 1rem;
+  font-style: italic;
+  border-left: none;
+}
+figure.captioned {
+  width: 60%;
+  padding: 5px;
+  margin-left: auto;
+  margin-right: auto;
+}
+img.captioned {
+  max-width: 90%;
+  margin-left: auto;
+  margin-right: auto;
+}
+figurecaption.captioned {
+  font-size: 70%;
+  font-style: italic;
+  text-align: center;
+}
+.icon_column {
+  float: left;
+  width: 33.3%;
+  padding: 5px;
+}
+.icon_row::after {
+  content: "";
+  clear: both;
+  display: table;
+}
+</style>
+
+<div style="margin-left:auto; margin-right: auto; width: 95%; margin: 5px; padding: 20px; font-size: 90%">
+<i>Below is a statement delivered November 2018 to the Euro-Bioimaging Industry
+Board regarding the support of proprietary file formats by Bio-Formats. This
+was discussed during the</i><a href="https://www.embl.de/training/events/2018/IMJ18-01/">
+From Images to Knowledge with ImageJ & Friends</a> <i>meeting in
+December and since then, there have been a growing number of conversations
+about a common format for bioimaging data. We're posting it here to tie the
+conversations back together and continue an open discussion of this critical
+issue.</i>
+</div>
+
+As many of you know, work on Bio-Formats began in 2006, and over the first 10
+years of development, support was added for over 140 file formats. If you
+include the per-format variants that have emerged over the years, that might be
+as much as 5 or 10 times higher, but precise numbers are difficult at best.
+
+{% include caption.html
+    file="2019/growth-formats.png"
+    description="Growth of file format readers (2006-2018)"
+    class="captioned" %}
+
+In 2016, we issued [a public
+statement](https://blog.openmicroscopy.org/file-formats/community/2016/01/06/format-support/)
+that OME, or more specifically its funding model, was not going to keep up with
+the accelerated development of new formats. We warned that we would be spending
+less time on closed formats, and we suggested that format developers either
+move to open formats or invest their own time or money to support their
```

Comment:
sentence seems to end abruptly

Reply:
Yeah, there's also one xxx. I'll finalize the text over the weekend, but I wanted to get feedback on the CSS (so I'd stop making changes)